src/import/ensureConfiguration.ts

```diff
@@ -453,26 +453,32 @@ function getMenuObjectFromMenuXML(menuXML: string): YAMLConfigurationMenuTree {
         if (li.a && !Array.isArray(li.a)) {
           const name = li.a._text;
           const link = li.a._attributes?.href;
+          const inNewTab = li.a._attributes?.target === '_blank' ? 'new-tab ' : '';
+          const isExternal = li.a.img?._attributes?.src === 'external.png' ? 'external ' : '';
+          const menuLinkWithKeywords = `${inNewTab}${isExternal}${link}`;
```

Comment:
If you make functions instead, you can re-use them in the sub-menu too and have less duplicate code.  E.g.,
```ts
const inNewTab = (item: any): string => item.a._attributes?.target === '_blank' ? 'new-tab ' : '';
const isExternal = (item: any): string => item.a.img?._attributes?.src === 'external.png' ? 'external ' : '';
const menuLinkWithKeywords = `${inNewTab(li)}${isExternal(li)}${link}`;
```
_NOTE: I did `item: any` because I was too lazy to lookup what the real type is..._

Reply:
That's a good idea and I added the functions. `item: any` actually seems to be the real type we have specified for the list items.